apps/ex_shop/web/models/order.ex

```diff
@@ -4,33 +4,40 @@ defmodule ExShop.Order do
   alias __MODULE__
 
   schema "orders" do
+    # concrete fields
     field :slug, :string
     field :state, :string, default: "cart"
     field :total, :decimal, default: Decimal.new("0")
     field :confirmation_status, :boolean, default: true
     field :product_total, :decimal, default: Decimal.new("0")
 
+    # virtual fields
     field :confirm, :boolean, virtual: true
     field :tax_confirm, :boolean, virtual: true
-
+    field :same_as_shipping, :boolean, virtual: true
```

Comment:
Can we please change to same_as_billing and ask for Billing Address all the time and save some typing for shipping_address in cases, where they are same ?

Invoice would be needed in all the cases whether shipped or not ?